adafruit_displayio_ssd1306.py

```diff
@@ -105,13 +105,13 @@ def sleep(self):
         active prior to sleeping. MP can access (update) the built-in display RAM.
         """
         if self._is_awake:
-            self.bus.send(int(0xAE), "")  # 0xAE = display off, sleep mode
+            self.bus.send(int(0xAE), b"")  # 0xAE = display off, sleep mode
```

Comment:
The `int()` is redundant.
```suggestion
            self.bus.send(0xAE, b"")  # 0xAE = display off, sleep mode
```